modules/ossm-config-sampling.adoc

```diff
@@ -5,18 +5,23 @@
 [id="ossm-config-sampling_{context}"]
 = Adjusting the sampling rate
 
-The distributed tracing sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces.
+A trace is an execution path between services in the service mesh. A trace is comprised of one or more spans. A span is a logical unit of work that has a name, start time, and duration. The sampling rate determines how often a trace is generated.
 
-A trace is an execution path between services in the service mesh. A trace is comprised of one or more spans. A span is a logical unit of work that has a name, start time, and duration.
+The Envoy proxy sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces. For example, set `spec.tracing.sampling` to `100.`
```

Comment:
does the example make sense? I think it should be lower than 100 because the 100 is the default.

Reply:
I thought that `100` denotes 100% :)

Reply:
I updated this section based on Julie's suggestion.

CC: @JStickler, @pavolloffay

Reply:
@pavolloffay that's what I figured you were thinking.  =)